content/_partners_impact_stories/dol._en.md

```diff
@@ -21,7 +21,7 @@ challenge: >-
     For many families, unemployment insurance (UI) can provide critical financial support, helping to pay for groceries or utilities. The COVID-19 pandemic brought this into sharp focus, especially the need for unemployment benefit recipients to receive their benefits in a timely, accessible way. At that time, many state-level UI offices were closed, presenting an additional barrier to claimants.
 
 
-    As states increased the use of online access to file for UI benefits, the delivery of benefits exposed the vulnerability of online filing systems to identify fraud. States needed tools to secure equitable access to benefits while providing sufficient security to verify identity.
+    As states increased the use of online access to file for UI benefits, the delivery of benefits exposed the vulnerability of online filing systems to identify fraud. States needed tools to secure access to benefits while providing sufficient security to verify identity.
 
```

Comment:
It looks like there is mention of SBA below in the impact section, should that be fixed?

Reply:
confirmed

Reply:
I thought we had intended to remove SBA references altogether as of #1293 / LG-13591 ?

Reply:
Oops, yeah, that's what I was referencing. I didn't realize that we had mixed up Association/Administration.

Reply:
@mitchellhenke The initial change is more time sensitive, I believe, than the others and was split for easy approval :), if you're ready to approve.

Reply:
The second change that has been rolled into this is documented at the following locations:

https://cm-jira.usa.gov/browse/LG-15492

https://cm-jira.usa.gov/browse/LG-15587?focusedId=2169115&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2169115